circle.yml

```diff
@@ -12,9 +12,13 @@ dependencies:
     - echo "memory_limit = 512M" > ~/.phpenv/versions/$(phpenv global)/etc/conf.d/memory.ini
     # Disable mail sending. Sendmail is not working on Circle.
     - echo "sendmail_path = /bin/true" > ~/.phpenv/versions/$(phpenv global)/etc/conf.d/sendmail.ini
+    # Clean up existing global composer installation. It contains dependencies
+    # which do not match the PHP version we use. This prevents installation of
+    # other packages.
+    - rm -rf ~/.composer
     # Install drush globally using composer.
-    # Prefer source to get around GitHub API rate limits.
-    - composer global require drush/drush:6.*  --prefer-source --no-interaction
+    # Use prefer-source to get around GitHub API rate limits.
+    - composer global require drush/drush:6.* --prefer-source --no-interaction
```

Comment:
Agree with @guddo here. I do not think there is a reason for bumping this right now either.

We could consider switching to Travis to test the build using multiple versions of drush (and PHP and...) but that would have other consequences e.g. slower build times, more complex asset handling etc.